src/ui/utils/text.ts

```diff
@@ -0,0 +1,10 @@
+const ellipsisBetweenText = (text: string) => {
```

Comment:
No need to add one more file to our repo for such a small operation. Here is how we did that in the past:

![07052024093658](https://github.com/cardano-foundation/cf-identity-wallet/assets/119612231/84376e04-d94a-492f-96c8-f2121a1f93e8)